server-worker/stops/onzo.js

```diff
@@ -0,0 +1,35 @@
+const Request = require('request')
+
+const onzos = []
+
+const onzo = {
+  async getBikes(lat, dis, lon) {
+    // const { lat, dis, lon } = req.query
+
+    const options = {}
+    options.url = `https://app.onzo.co.nz/nearby/${lat}/${lon}/${dis}`
+    options.json = true
+    Request(options, (error, response, body) => {
+      if (error) {
+        return
+      }
+
+      await body.data.forEach(onzo => {
+        onzos.append({
+          stop_id: onzo.iccid,
```

Comment:
I would probably add a prefix like, "onzo-" to the stop id, so you can use the same route and figure out what is being queried there.